src/components/PivotTable/PivotTableEmptyCell.js

```diff
@@ -3,14 +3,15 @@ import PropTypes from 'prop-types'
 import { cell as cellStyle } from './styles/PivotTable.style'
 import { PivotTableCell } from './PivotTableCell'
 
-export const PivotTableEmptyCell = ({ type, ...props }) => {
+export const PivotTableEmptyCell = ({ type, forwardedRef, ...props }) => {
```

Comment:
Should this use `React.forwardRef` as well?

Reply:
Changed.